mitmproxy/addons/serverplayback.py

```diff
@@ -30,7 +30,15 @@ def load(self, loader):
             "server_replay_kill_extra",
             bool,
             False,
-            "Kill extra requests during replay (for which no replayable response was found).",
+            "Kill extra requests during replay (for which no replayable response was found)."
+            "[Deprecated, prefer to use server_replay_extra='kill']",
+        )
+        loader.add_option(
+            "server_replay_extra",
+            str,
+            "forward",
+            "Behaviour for extra requests during replay for which no replayable response was found.",
```

Comment:
Let's add a second sentence explaining what the numbers do. Something like that?
```
Setting a numeric string value will return an empty HTTP response with the respective status code.
```